ansible_ai_connect/ai/api/model_pipelines/llamastack/pipelines.py

```diff
@@ -146,27 +172,18 @@ def self_test(self) -> HealthCheckSummary:
                 MODEL_MESH_HEALTH_CHECK_MODELS: "ok",
             }
```

Comment:
You should probably change [L#169-L#174](https://github.com/ansible/ansible-ai-connect-service/blob/1437b87d2f9e8fe3b487d29405a56f3105bee899/ansible_ai_connect/ai/api/model_pipelines/llamastack/pipelines.py#L169-L174) to something like this:
```
summary: HealthCheckSummary = HealthCheckSummary(
  {
    MODEL_MESH_HEALTH_CHECK_PROVIDER: "llama-stack",
    MODEL_MESH_HEALTH_CHECK_MODELS: "ok",
    MODEL_MESH_HEALTH_CHECK_INDEX: "ok",
  }
)
```
i.e. one entry for the model/inference and another for the RAG index.